addons/website/views/snippets/s_features_wave.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_features_wave" name="Features Wave">
+    <section class="s_features_wave o_cc o_cc5 pt64 pb64" data-oe-shape-data="{'shape':'web_editor/Wavy/11', 'showOnMobile':true}">
+        <div class="o_we_shape o_web_editor_Wavy_11 o_shape_show_mobile"/>
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-12 pb-24">
+                    <h3 style="text-align: center;">Everything you need</h3>
+                    <p class="lead" style="text-align: center;">List and describe the key features of your solution or service.</p>
+                </div>
+                <div class="col-lg-4 pt24 pb24">
+                    <i class="fa fa-thumbs-o-up d-block mx-auto border rounded fa-2x" style="background-color: rgba(0, 0, 0, 0);"/>
+                    <br/>
+                    <h4 style="text-align: center;">Reliability</h4>
+                    <p style="text-align: center;">Consistent performance and uptime ensure efficient, reliable service with minimal interruptions and quick response times.</p>
+                </div>
+                <div class="col-lg-4 pt24 pb24">
+                    <i class="fa fa-star-o d-block mx-auto border rounded fa-2x" style="background-color: rgba(0, 0, 0, 0);"/>
+                    <br/>
+                    <h4 style="text-align: center;">Performance</h4>
+                    <p style="text-align: center;">Speed and efficiency ensure tasks are completed quickly and resources are used optimally, enhancing productivity and satisfaction.</p>
+                </div>
+                <div class="col-lg-4 pt24 pb24">
+                    <i class="fa fa-flag-o d-block mx-auto border rounded fa-2x" style="background-color: rgba(0, 0, 0, 0);"/>
+                    <br/>
+                    <h4 style="text-align: center;">Scalability</h4>
+                    <p style="text-align: center;">Growth capability is a system's ability to scale and adapt, meeting increasing demands and evolving needs for long-term success.</p>
+                </div>
+                <div class="col-lg-4 pt24 pb24">
+                    <i class="fa fa-edit d-block mx-auto border rounded fa-2x" style="background-color: rgba(0, 0, 0, 0);"/>
+                    <br/>
+                    <h4 style="text-align: center;">Customizable Settings</h4>
+                    <p style="text-align: center;">Tailor the platform to your needs, offering flexibility and control over your user experience.</p>
+                </div>
+                <div class="col-lg-4 pt24 pb24">
+                    <i class="fa fa-user-o d-block mx-auto border rounded fa-2x" style="background-color: rgba(0, 0, 0, 0);"/>
```

Comment:
Note: note sure if we should use `role="img"` or `role="presentation"`... let's keep an eye on [this thread](https://github.com/odoo/odoo/pull/166138#discussion_r1679301377) while the PR proceed further...

Reply:
added `role="presentation"` 👍